examples/vision/simsiam.py

```diff
@@ -90,36 +94,42 @@
 """
 
 
+strength=[0.4, 0.4, 0.3, 0.1]
+
+random_flip = layers.RandomFlip(mode="horizontal_and_vertical")
+random_crop = layers.RandomCrop(CROP_TO, CROP_TO)
+random_brightness=layers.RandomBrightness(0.8 * strength[0])
+random_contrast=layers.RandomContrast((1 - 0.8 * strength[1],1 + 0.8 * strength[1]))
+random_saturation=keras_cv.layers.RandomSaturation((0.5 - 0.8 * strength[2], 0.5 + 0.8 * strength[2]))
+random_hue=keras_cv.layers.RandomHue(0.2 * strength[3],[0,255])
+grayscale=keras_cv.layers.Grayscale()
+
 def flip_random_crop(image):
     # With random crops we also apply horizontal flipping.
-    image = tf.image.random_flip_left_right(image)
-    image = tf.image.random_crop(image, (CROP_TO, CROP_TO, 3))
+    image = random_flip(image)
+    image = random_crop(image)
     return image
 
 
-def color_jitter(x, strength=[0.4, 0.4, 0.4, 0.1]):
-    x = tf.image.random_brightness(x, max_delta=0.8 * strength[0])
-    x = tf.image.random_contrast(
-        x, lower=1 - 0.8 * strength[1], upper=1 + 0.8 * strength[1]
-    )
-    x = tf.image.random_saturation(
-        x, lower=1 - 0.8 * strength[2], upper=1 + 0.8 * strength[2]
-    )
-    x = tf.image.random_hue(x, max_delta=0.2 * strength[3])
+def color_jitter(x):
+    x = random_brightness(x)
+    x = random_contrast(x)
+    x = random_saturation(x)
+    x = random_hue(x)
     # Affine transformations can disturb the natural range of
     # RGB images, hence this is needed.
-    x = tf.clip_by_value(x, 0, 255)
+    x = ops.clip(x, 0, 255)
     return x
 
 
 def color_drop(x):
-    x = tf.image.rgb_to_grayscale(x)
-    x = tf.tile(x, [1, 1, 3])
+    x = grayscale(x)
+    x = ops.tile(x, [1, 1, 3])
     return x
 
 
 def random_apply(func, x, p):
-    if tf.random.uniform([], minval=0, maxval=1) < p:
+    if keras.random.uniform([], minval=0, maxval=1,seed=152) < p:
```

Comment:
Remove the `seed` argument, otherwise you'll get the same output at every call

Reply:
Sure, I have made the necessary changes!